content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -490,3 +487,5 @@ kubectl delete pod security-context-demo-4
 * [Pod Security Policies](/docs/concepts/policy/pod-security-policy/)
 * [AllowPrivilegeEscalation design
   document](https://git.k8s.io/community/contributors/design-proposals/auth/no-new-privs.md)
+* For more information about security mechanisms in Linux, see
+[Overview of Linux Kernel Security Features](https://www.linux.com/learn/overview-linux-kernel-security-features) (Note: Some information is out of date)
```

Comment:
nit: indents for the list items in Markdown
- This does not make any changes to the page displayed.
- If you apply this, please consider to squash into one single commit.

```suggestion
* For more information about security mechanisms in Linux, see
  [Overview of Linux Kernel Security Features](https://www.linux.com/learn/overview-linux-kernel-security-features) (Note: Some information is out of date)
```